backend/src/app/util/time.clj

```diff
@@ -158,6 +158,7 @@
      :iso8601 (Instant/from (.parse DateTimeFormatter/ISO_INSTANT ^String s)))))
 
 (defn is-after?
+  "Analgous to: d1 > d2"
```

Comment:
Very minor, but it't better to respect the name of the parameters: da > db

Reply:
true